### PR TITLE
Feat/badge

### PR DESCRIPTION
**Pull request by k-rahul21**

### CODE REVIEW

**Comment by AshwiniSetty** [comment]
The code looks good to be merged.

**Review by AshwiniSetty — COMMENTED** [review]
The code looks good to be merged.

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  margin: 0 1rem;
+}
+
+.badge-number {
+  font-size: 0.9rem;
+  background-color: var(--secondary-color);
+  border-radius: 50%;
+  color: var(--primary-color);
+  height: 25px;
+  width: 25px;
```
Comment (znjs) [inline components/badge/badge.css]: Suggestion: If possible try using relative units

**Comment by znjs** [comment]
The main content not occupying the whole space for some reason, do check once
image

**Pull request merged**